tests/axiosInterceptor.test.js

```diff
@@ -0,0 +1,94 @@
+jest.setTimeout(60000)
+
+import StoryblokClient from '../source/index'
+
+describe('Client should accept response interceptor as a function', () => {
+  const Storyblok = new StoryblokClient({
+    accessToken: 'trB5kgOeDD22QJQDdPNCjAtt',
+    cache: { type: 'memory', clear: 'auto' },
+    responseInterceptor: (res) => {
+      return res
+    },
+  })
+  it('should RESPONSE to be TRUTHY', async () => {
+    await Storyblok.getAll('cdn/links')
+    expect(Storyblok.client.interceptors.response.handlers.length).toBeTruthy()
+  })
+  it('should REQUEST to be FALSY', async () => {
+    await Storyblok.getAll('cdn/links')
+    expect(Storyblok.client.interceptors.request.handlers.length).toBeFalsy()
+  })
+})
+
+describe('Client should be initialized without interceptors', () => {
+  const Storyblok = new StoryblokClient({
+    accessToken: 'trB5kgOeDD22QJQDdPNCjAtt',
+    cache: { type: 'memory', clear: 'auto' },
+  })
+  it('should RESPONSE to be FALSY', async () => {
+    await Storyblok.getAll('cdn/links')
+    expect(Storyblok.client.interceptors.response.handlers.length).toBeFalsy()
+  })
+  it('should REQUEST to be FALSY', async () => {
+    await Storyblok.getAll('cdn/links')
+    expect(Storyblok.client.interceptors.request.handlers.length).toBeFalsy()
+  })
+})
+
+describe('Client should throw an error if NO RESPONSE is returned by the responseInterceptor function', () => {
+  it('should trhow a TypeError if NOTHING is returned', async () => {
+    const Storyblok = new StoryblokClient({
+      accessToken: 'trB5kgOeDD22QJQDdPNCjAtt',
+      cache: { type: 'memory', clear: 'auto' },
+      responseInterceptor: () => {
+      },
+    })
+    let error = ''
+    try {
+      await Storyblok.getAll('cdn/links')
+    } catch(err) {
+      error = err
+    }
+    expect(error).toEqual(new TypeError('Cannot read property \'data\' of undefined'));
+  })
+  it('should trhow a TypeError if NULL is returned', async () => {
```

Comment:
A typo in this test title, `should trhow...` should be `should throw...`

Reply:
Thanks! Will correct now!